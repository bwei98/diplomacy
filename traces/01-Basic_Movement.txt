# Spring 1901
F Lon - NTH; A Lvp - Yor; F Edi - NWG
A Mar - Bur; A Par - Pic; F Bre - MAO
A Mun - Ruh; F Kie - Den; A Ber - Sil
A Mos - Ukr; A War - Gal; F Sev - BLA; F Stp - Fin
A Vie - Tyr; F Tri - ADR; A Bud - Ser
A Rom - Apu; A Ven - Pie; F Nap - ION
A Con - Bul; A Smy - Syr; F Ank - Arm
#
# END