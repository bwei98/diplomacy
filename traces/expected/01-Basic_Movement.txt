# Spring 1901
F NTH; A Yor; F NWG
Lvp; Edi; Lon
F MAO; A Pic; A Bur
Par; Bre; Mar
F Den; A Sil; A Ruh
Kie; Ber; Mun
F Fin; A Ukr; A Gal; F BLA
Mos; Stp; Sev; War
F ADR; A Ser; A Tyr
Vie; Tri; Bud
A Apu; A Pie; F ION
Rom; Nap; Ven
A Bul; A Syr; F Arm
Con; Smy; Ank